packages/info/lib/doctor.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const define = require('mixinable');
+
+const {
+  sync: { callable: callable, sequence },
+} = define;
+
+const strategies = {
+  diagnoseConfig: callable,
+  diagnoseDuplicatePackages: callable,
+  diagnoseInvalidPackages: callable,
```

Comment:
So only the doctor can do the diagnose? (I'm asking because the strategy is callable or a.k.a override).

And do we also want to provide a check to make sure all packages of a specific workspace (like `@untool/*` or `hops-*`) have the same version in common?

Reply:
There is no way for anything to actually override strategies/implementations in this context, so yes, it is only this very implementation.

I would very much like to add a check like the one you describe, but maybe after merging this PR?